includes/class-plugin.php

```diff
@@ -101,6 +101,8 @@ public function add_actions_and_filters() {
         add_action( 'admin_enqueue_scripts', [ $this, 'enqueue_admin_scripts' ] );
         add_action( 'admin_enqueue_scripts', [ $this, 'enqueue_redis_metrics' ] );
 
+        add_action( 'admin_bar_menu', [ $this, 'render_admin_bar' ], 998 );
```

Comment:
PR looks great. Why the high priority?

Reply:
Taken from the original issue: https://github.com/rhubarbgroup/redis-cache/issues/224

> Add with priority of 998, just before Query Monitor?